ex1/functions.py

```diff
@@ -46,15 +46,97 @@ def f():
 ##############################
 
 # Define a function named add that takes two numbers and returns the sum.
+def add(a,b):
+	"""Returns the sum of two given numbers.
+	
+	Adds variables a and b an returns the result:
+	
+	Parameters
+	----------
+	a : int	
+		First number to be added to.
+	b : int	
+		Number added to a.
+	
+	Returns
+	-------
+	sum : int
+		Result of addition
+	"""
+	
+	sum = a + b
+	return sum
 
 # Define a function named to_tuple that takes three arguments and returns a
 # tuple of these three arguments.
+def to_tuple(arg1, arg2, arg3):
+	"""Returns a tuple of arguments
+	
+	Takes 3 arguments and returns a tupel.
+	
+	Parameters
+	----------
+	arg1, arg2, arg3 : optional
+		Arguments formed into a tupel.
+		
+	Returns
+	-------
+	tuple : tuple
+		Tuple of all the arguments
+	"""
+	tuple = (arg1, arg2, arg3)
```

Comment:
No need to put this into a tuple manually since multiple return values are automatically packed into a tuple.